flask_caching/backends/rediscache.py

```diff
@@ -273,3 +273,57 @@ def __init__(
         self._read_clients = sentinel.slave_for(master)
 
         self.key_prefix = key_prefix or ""
+        
+class RedisClusterCache(RedisCache):
+    """Uses the Redis key-value store as a cache backend.
+
+    The first argument can be either a string denoting address of the Redis
+    server or an object resembling an instance of a rediscluster.RedisCluster class.
+
+    Note: Python Redis API already takes care of encoding unicode strings on
+    the fly.
+
+
+    :param cluster: The redis cluster nodes address separated by comma.
+                    e.g. host1:port,host2:port2,host3:port3 .
+    :param password: password authentication for the Redis server.
+    :param default_timeout: the default timeout that is used if no timeout is
+                            specified on :meth:`~BaseCache.set`. A timeout of
+                            0 indicates that the cache never expires.
+    :param key_prefix: A prefix that should be added to all keys.
+
+    Any additional keyword arguments will be passed to
+    ``rediscluster.RedisCluster``.
+    """
+    def __init__(self,
+                 cluster="",
+                 password="",
+                 default_timeout=300,
+                 key_prefix="",
+                 **kwargs):
+        super().__init__(default_timeout=default_timeout)
+
+        if kwargs.get("decode_responses", None):
+            raise ValueError("decode_responses is not supported by "
+                             "RedisCache.")
+
+        try:
+            from rediscluster import RedisCluster
+        except ImportError:
+            raise RuntimeError("no rediscluster module found")
+        HOST, PORT = 0, 1
```

Comment:
while having magic numbers is generally not a good idea - however here it would be more readable when using just node[0].strip() and node[1].strip() in my opinion

Reply:
Yeah, Magic numbers is not elegant code usually, I like to make it searchable by macro definitions. It’s also good to align the code style here, so I update these code.